gandalf.js

```diff
@@ -14,114 +14,117 @@
 ( function( document, window ) {
```

Comment:
SCRIPT-Consider renaming this variable to something more descriptive for better readability.